src/vs/workbench/contrib/tasks/electron-browser/task.contribution.ts

```diff
@@ -2195,16 +2210,33 @@ class TaskService extends Disposable implements ITaskService {
 		if (!this.canRunCommand()) {
 			return;
 		}
+		if (arg === 'terminateAll') {
+			this.terminateAll();
+			return;
+		}
 		let runQuickPick = (promise?: Promise<Task[]>) => {
 			this.showQuickPick(promise || this.getActiveTasks(),
 				nls.localize('TaskService.tastToTerminate', 'Select task to terminate'),
 				{
 					label: nls.localize('TaskService.noTaskRunning', 'No task is currently running'),
-					task: null
+					task: undefined
 				},
-				false, true
-			).then(task => {
+				false, true,
+				undefined,
+				[{
```

Comment:
Having the All running tasks option show in the quick pick when there is only one task is a bit confusing. We should only show it when there is > 1 task.